ios/PurchasesHybridCommon/PurchasesHybridCommonUI/PaywallProxy.swift

```diff
@@ -140,20 +193,32 @@ import UIKit
 
     private func privatePresentPaywall(displayCloseButton: Bool = false,
                                        content: Content = .defaultOffering,
+                                       fontName: String? = nil,
                                        paywallResultHandler: ((String) -> Void)? = nil) {
         guard let rootController = Self.rootViewController else {
             NSLog("Unable to find root UIViewController")
             return
         }
 
+        let fontProvider: PaywallFontProvider
+        if let fontName = fontName {
+            fontProvider = CustomPaywallFontProvider(fontName: fontName)
+        } else {
+            fontProvider = DefaultPaywallFontProvider()
+        }
+
         let controller: PaywallViewController
         switch content {
         case let .offering(offering):
-            controller = PaywallViewController(offering: offering, displayCloseButton: displayCloseButton)
+            controller = PaywallViewController(offering: offering, 
+                                               fonts: fontProvider,
+                                               displayCloseButton: displayCloseButton)
         case let .offeringIdentifier(identifier):
-            controller = PaywallViewController(offeringIdentifier: identifier, displayCloseButton: displayCloseButton)
+            controller = PaywallViewController(offeringIdentifier: identifier, 
+                                               fonts: fontProvider,
+                                               displayCloseButton: displayCloseButton)
         case .defaultOffering:
-            controller = PaywallViewController(displayCloseButton: displayCloseButton)
+            controller = PaywallViewController(fonts: fontProvider, displayCloseButton: displayCloseButton)
```

Comment:
Tried by calling `updateWith(font` but it breaks sizing in React Native so I pass this in the constructor